page/layouts/shortcodes/faq.html

```diff
@@ -0,0 +1,37 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
+<div class="expand">
+    <div class="expand-label" style="cursor: pointer;" onclick="$h = $(this);$h.next('div').slideToggle(100,function () {$h.children('i').attr('class',function () {return $h.next('div').is(':visible') ? 'fas fa-chevron-down' : 'fas fa-chevron-right';});});">
+        <i style="font-size:x-small;" class="fas fa-chevron-right"></i>
+        <span>
+        {{$expandMessage := T "Expand-title"}}
+    	{{ if .IsNamedParams }}
+    	{{.Get "default" | default $expandMessage}}
+    	{{else}}
+    	{{.Get 0 | default $expandMessage}}
+    	{{end}}
+    	</span>
+    </div>
+    <div class="expand-content" style="display: none;">
+        {{.Inner | safeHTML}}
+    </div>
+</div>
+
+<script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "FAQPage",
+      "mainEntity": [{
+        "@type": "Question",
+        "name": "{{$expandMessage := T "Expand-title"}}
+            {{ if .IsNamedParams }}
+            {{.Get "default" | default $expandMessage}}
+            {{else}}
+            {{.Get 0 | default $expandMessage}}
+    	    {{end}}",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "{{.Inner | safeHTML}}"
```

Comment:
This will produce HTML tags within `text`. Is that valid?

Reply:
not sure, if thats valid for "text". 
but that seams to be fine for the hugo default expand.html shortcode.

Also the result / content of the expand / faq should be able to contain html.

Reply:
> but that seams to be fine for the hugo default expand.html shortcode.

We are discussing the JSON-LD content here, not the regular content.